.github/workflows/build_wheels_linux_reusable.yml

```diff
@@ -3,26 +3,59 @@ name: Build Linux Wheels
 on:
   workflow_call:
     inputs:
-      domain:
-        description: 'name of the domain library calling this reusable workflow.'
+      repository:
+        description: 'name of the domain respository calling this reusable workflow.'
+        required: true
+        type: string
+      docker_image:
+        description: 'docker image container for running the reusable workflow.'
+        required: true
+        type: string
+      python_version:
+        description: 'python version to use for building wheels binary'
+        required: true
+        type: string
+      gpu_arch_version:
+        description: 'cuda version to use for building wheels binary'
         required: true
         type: string
 
 jobs:
   wheels:
     runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: ${{ inputs.python_version }}
+      PACKAGE_TYPE: wheel
+      REPOSITORY: ${{ inputs.repository }}
+      GPU_ARCH_VERSION: ${{ inputs.gpu_arch_version }}
+    container:
+      image: ${{ inputs.docker_image }}
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.x
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.x'
-    - name: Install dependencies
-      run: python -m pip install --upgrade setuptools wheel
-      # Include domain-specific pre-build step here 
-      # (where each domain lib brings its own setup script)
-      # Use the domain input to call the right composite action
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+    - name: Set artifact name
+      run: |
+        echo "ARTIFACT_NAME=${REPOSITORY/\//_}_${REF}_${PYTHON_VERSION}" >> "${GITHUB_ENV}"
+    - name: Install pytorch-pkg-helpers
+      run: |
+        python3 -m pip install pytorch-pkg-helpers==0.1.0
```

Comment:
this 0.1.0 is no longer available. Should we change to 0.1.1?

Reply:
uh, looks like 0.1.1 encounters same issue.

Reply:
python3 -m pip install pytorch-pkg-helpers or python3 -m pip install pytorch-pkg-helpers==0.1.2 may work.